networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/network/config/NetworkConfig.java

```diff
@@ -362,9 +362,9 @@ private void validateAddresses(final List<String> addresses, final String cliOpt
       }
     }
 
-    private void validatePort(final int port, final String cliOption) {
+    private void validatePort(final int port, final String parameter) {
```

Comment:
Not sure on this change, we still have cliOptions in `validateAddresses`

Reply:
Yeah, I like cliOption better to be honest. Changed.